python/paddle/fluid/contrib/slim/tests/quantization/config_mkldnn_int8.yaml

```diff
@@ -0,0 +1,11 @@
+version: 1.0
+strategies:
+    mkldnn_post_training_strategy:
+        class: 'MKLDNNPostTrainingQuantStrategy'
+        int8_model_save_path: 'OUTPUT_PATH'
+        fp32_model_path: 'MODEL_PATH'
```

Comment:
Do you need `cpu_math_library_num_threads` here, i,e do not directly use `FLAGS_omp_num_thread` to change it? @bingyanghuang @wojtuss 
Maybe you can update in next PR.

Reply:
@luotao1 , I think you are right, the number of threads could be set here instead of using the `FLAGS_OMP_NUM_THREADS` environment variable.